src/main/java/pl/javastart/task/UniversityApp.java

```diff
@@ -91,7 +171,16 @@ public void addGrade(int studentIndex, String groupCode, double grade) {
      *
      * @param index - numer indesku studenta dla którego wyświetlić oceny
      */
+
     public void printGradesForStudent(int index) {
+        double[] gradeSum = new double[groupCount];
+        for (int i = 0; i < groupCount; i++) {
+            for (int j = 0; j < groups[i].getStudentsCounter(); j++) {
+                if (groups[i].students[j].getIndex() == index)
+                    gradeSum[i] += groups[i].students[j].getGrade();
+            }
+            System.out.println(groups[i].getName() + ": " + gradeSum[i]);
+        }
```

Comment:
Gdybyś wprowadził dodatkową klasę Grade i tablicę `Grade[] grades`, to logika tutaj sprowadzałaby się do
```
for (Grade grade: grades) {
  if (grade.getStudent().getIndex() == index) {
      ...wyświetl ocenę studenta
  }
}
```